nixos/modules/config/users-groups.nix

```diff
@@ -120,7 +120,10 @@ let
       shell = mkOption {
         type = types.str;
         default = "/run/current-system/sw/bin/nologin";
-        description = "The path to the user's shell.";
+        description = ''
+          The path to the user's shell. Can use store paths,
+          like <literal>${pkgs.bash}/bin/bash</literal>.
```

Comment:
Shouldn't this be `''${pkgs.bash}...` to avoid expansion of the `pkgs.bash` store path?